### ISSUE TITLE
Documentation regarding neural net-compatible approaches

### ISSUE THREAD

**Issue by scottcode** [interpretml/interpret#101]
First, thanks to everyone who is building `interpret`. It looks live a valuable package for pulling together model interpretation methods into a single package. 

The only documentation I see is the `README.md`, and from that it's hard for me to tell what methods `interpret` has that might help with deep learning models. Presumably the blackbox portion of the package is most applicable, but it's hard to tell from the `README` what all the methods for blackbox are. In particular, does `interpret` or any of its dependencies include any utilities for performing the Integrated Gradients approach to explaining neural networks? ([paper](https://arxiv.org/abs/1703.01365), [repo](https://github.com/ankurtaly/Integrated-Gradients))

**Comment by interpret-ml** [comment]
Hi @scottcode, 

Thanks for the note -- we definitely need to be more clear about which methods are currently integrated. 

You're right that the [blackbox portion](https://nbviewer.jupyter.org/github/interpretml/interpret/blob/master/examples/python/notebooks/Explaining%20Blackbox%20Classifiers.ipynb) of the package is most applicable at the moment, but these methods will work on any system, not just deep learning models. 

If you're immediately looking for neural-network specific approaches, we'd recommend looking at [DeepSHAP](https://github.com/slundberg/shap) and the [PyTorch Captum](https://github.com/pytorch/captum) libraries. In particular, Captum directly supports the [Integrated Gradients approach](https://github.com/pytorch/captum/blob/89df6917f9f970c8eece3e50bdad2add4b2ac77d/captum/attr/_core/integrated_gradients.py#L21).

Thanks!
-InterpretML Team

**Comment by scottcode** [comment]
Thanks for the pointers!
